Comando,Posição,Direção,Sensor Esquerdo,Sensor Frente,Sensor Direito,Compartimento
LIGAR,"(0, 3)",2,PAREDE,VAZIO,PAREDE,SEM CARGA
INFO,"(0, 3)",2,"Posicao inicial: (0, 3), Direcao inicial: 2",SEM CARGA
INFO,"(0, 3)",2,Iniciando busca pelo humano,SEM CARGA
INFO,"(0, 3)",2,"Iniciando execução do caminho: [(0, 3), (1, 3), (1, 2), (1, 1), (2, 1), (3, 1), (3, 2), (3, 3), (3, 4), (3, 5), (4, 5), (5, 5), (6, 5), (7, 5)]",SEM CARGA
ERRO,"(0, 3)",2,"Erro ao executar comando A: Colisao com parede! Tentativa de mover de (0, 3) para (0, 4)",SEM CARGA
ERRO,"(0, 3)",2,"Colisao com parede! Tentativa de mover de (0, 3) para (0, 4)","Traceback (most recent call last):
  File ""C:\Users\Victus\Desktop\Segundo Semestre 2024\Serviços Cognitivos\new_ads1\robo_resgate.py"", line 32, in executar_resgate
    self.buscar_humano()
  File ""C:\Users\Victus\Desktop\Segundo Semestre 2024\Serviços Cognitivos\new_ads1\robo_resgate.py"", line 65, in buscar_humano
    self.executar_caminho(caminho)
  File ""C:\Users\Victus\Desktop\Segundo Semestre 2024\Serviços Cognitivos\new_ads1\robo_resgate.py"", line 97, in executar_caminho
    self.executar_comando('A')
  File ""C:\Users\Victus\Desktop\Segundo Semestre 2024\Serviços Cognitivos\new_ads1\robo_resgate.py"", line 139, in executar_comando
    self.simulador.mover_robo(comando)
  File ""C:\Users\Victus\Desktop\Segundo Semestre 2024\Serviços Cognitivos\new_ads1\simulador_labirinto.py"", line 72, in mover_robo
    raise ValueError(f""Colisao com parede! Tentativa de mover de {self.posicao_robo} para ({nova_i}, {nova_j})"")
ValueError: Colisao com parede! Tentativa de mover de (0, 3) para (0, 4)
",SEM CARGA
ERRO_FATAL,"(0, 3)",2,"Colisao com parede! Tentativa de mover de (0, 3) para (0, 4)","Traceback (most recent call last):
  File ""C:\Users\Victus\Desktop\Segundo Semestre 2024\Serviços Cognitivos\new_ads1\main.py"", line 24, in main
    robo.executar_resgate()
  File ""C:\Users\Victus\Desktop\Segundo Semestre 2024\Serviços Cognitivos\new_ads1\robo_resgate.py"", line 32, in executar_resgate
    self.buscar_humano()
  File ""C:\Users\Victus\Desktop\Segundo Semestre 2024\Serviços Cognitivos\new_ads1\robo_resgate.py"", line 65, in buscar_humano
    self.executar_caminho(caminho)
  File ""C:\Users\Victus\Desktop\Segundo Semestre 2024\Serviços Cognitivos\new_ads1\robo_resgate.py"", line 97, in executar_caminho
    self.executar_comando('A')
  File ""C:\Users\Victus\Desktop\Segundo Semestre 2024\Serviços Cognitivos\new_ads1\robo_resgate.py"", line 139, in executar_comando
    self.simulador.mover_robo(comando)
  File ""C:\Users\Victus\Desktop\Segundo Semestre 2024\Serviços Cognitivos\new_ads1\simulador_labirinto.py"", line 72, in mover_robo
    raise ValueError(f""Colisao com parede! Tentativa de mover de {self.posicao_robo} para ({nova_i}, {nova_j})"")
ValueError: Colisao com parede! Tentativa de mover de (0, 3) para (0, 4)
",SEM CARGA
